packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -255,14 +305,43 @@ export class Coachmark extends BaseComponent<ICoachmarkTypes, ICoachmarkState> {
           this.forceUpdate();
         }
 
+        document.addEventListener('keydown', this._onKeyDown, true);
```

Comment:
BaseComponent provides `_events` that should be used for these.. it automatically takes care of a lot of housekeeping for you. For example in [KeytipLayer.base.tsx](https://github.com/OfficeDev/office-ui-fabric-react/blob/74baa0a93fe133722769a69fef7ce72d9bc46dce/packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx#L134):
```
this._events.on(window, 'keydown', this._onKeyDown, true);
```